utils/cliutils/commandsflags.go

```diff
@@ -855,6 +856,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  transitive,
 		Usage: "[Default: false] Set to true to look for artifacts also in remote repositories. The search will run on the first five remote repositories within the virtual repository. Available on Artifactory version 7.17.0 or higher.` `",
 	},
+	searchInclude: cli.StringFlag{
+		Name:  searchInclude,
+		Usage: "[Optional] List of properties in the form of \"key1=value1;key2=value2,...\". Only fields that specified will be returned` ` ",
```

Comment:
* I'm pretty sure this flag description wouldn't help the users understand how to sue it. In addition to a clearer description, let's also include a link to the AQL documentation section, which specified the supported fields. 
* Let's also update the online documentation with the new few flag, and add an example.